src/com/owncloud/android/files/services/FileDownloader.java

```diff
@@ -80,6 +86,10 @@
     public static final String EXTRA_LINKED_TO_PATH = "LINKED_TO";
     public static final String ACCOUNT_NAME = "ACCOUNT_NAME";
 
+    public static final String KEY_FILE_REMOTE_PATH = "FILE_REMOTE_PATH";
+    public static final String KEY_ACCOUNT_NAME = "ACCOUNT_NAME";
```

Comment:
I'd rather call it ```EXTRA_ACCOUNT_NAME``` , to follow the same convention as other keys used for ```extras```  in ```Bundle``` .

Besides, since ```FileDownloader```  does not really need this string for itself, I would declare it in ```RetryDownloadJobService``` instead of here.